bigbluebutton-html5/imports/ui/components/chat/container.jsx

```diff
@@ -56,24 +73,6 @@ export default injectIntl(createContainer(({ params, intl }) => {
 
       title = intl.formatMessage(intlMessages.titlePrivate, { 0: chatUser.name });
       chatName = chatUser.name;
-
-      if (!chatUser.isOnline) {
-        const time = Date.now();
-        const id = `partner-disconnected-${time}`;
-        const messagePartnerLoggedOut = {
```

Comment:
I'm very confused now. Double-checked, and yeah.. we don't have the message "xxx left the meeting" anymore here.
It was silently removed here.

@antobinary confirm please, don't we need this message anymore?

![notification_left_the_meeting](https://user-images.githubusercontent.com/9299997/33186423-7d03113a-d057-11e7-952f-87f3d03f8d67.JPG)

Reply:
@OZhurbenko from what I know we still need this `X has left the meeting`.